integration-tests/api/__tests__/admin/order/order.js

```diff
@@ -2443,9 +2443,50 @@ describe("/admin/orders", () => {
       )
 
       expect(order.status).toEqual(200)
+      // id + totals + region relation
+      expect(Object.keys(order.data.order)).toHaveLength(12)
       expect(order.data.order).toEqual(
         expect.objectContaining({
           id: "test-order",
+          region: expect.any(Object),
+        })
+      )
+    })
+
+    it("retrieves an order with expand returnable_items only should return the entire object and only returnable_items as relation", async () => {
+      const api = useApi()
+
+      const order = await api.get(
+        `/admin/orders/${testOrderId}?expand=returnable_items`,
+        adminReqConfig
+      )
+
+      expect(order.status).toEqual(200)
+      // all order properties + totals + returnable_items relation
+      expect(Object.keys(order.data.order)).toHaveLength(29)
+      expect(order.data.order).toEqual(
+        expect.objectContaining({
+          id: "test-order",
+          returnable_items: [],
+        })
+      )
+    })
+
+    it("retrieves an order with expand returnable_items and field id should return the id and the retunable_items", async () => {
+      const api = useApi()
+
+      const order = await api.get(
+        `/admin/orders/${testOrderId}?expand=returnable_items&fields=id`,
+        adminReqConfig
+      )
+
+      expect(order.status).toEqual(200)
+      // id + totals + returnable_items relation
+      expect(Object.keys(order.data.order)).toHaveLength(12)
```

Comment:
question: Could we match the exact keys instead to make it even more robust?

Reply:
Yes sure 😊